test/server/util/twilio.js

```diff
@@ -18,6 +17,14 @@ describe('Twilio client methods', function() {
     expect(mutateCallConnectContact('CAcbbf06f666c72c51c59200de56ae54ff', idCollection)).to.deep.equal(returnUpdate('CAcbbf06f666c72c51c59200de56ae54ff', CONNECT_CONTACT_URL));
     done();
   });
+  it('should create a new call', (done) => {
+    expect(callVolunteer(1, 1, '+15555555555')).to.deep.equal(createCall());
+    done();
+  });
+  it('should return a call object', (done) => {
+    expect(typeof callVolunteer(1, 1, '+15555555555')).to.equal('object');
```

Comment:
we test that we get an object back, but how do we know it's a `call` object? what characteristics (or properties) does a call object have?

Reply:
Good point. We can expect it to have a callSID maybe, the to should match the dialled number, and that should be enough I think.

Reply:
Added another check to ensure certain keys are present.

Reply:
And another to test the dialed number is equal to "to"